### PR TITLE
replicate crio presubmits using kubetest2

### PR DESCRIPTION
**Pull request by elieser1101**
This is part of an effort to start using kubetest2 for crio e2e node tests. Being tracked in https://github.com/kubernetes/test-infra/issues/32567. starting with presubmits

### CODE REVIEW

**Comment by kannon92** [comment]
can you combine the presubmits into a single PR?

**Review by krzyzacy — APPROVED** [review]
/ok-to-test

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -1346,6 +1346,59 @@ presubmits:
         env:
         - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
           value: "1"
+- name: pull-kubernetes-node-crio-cgrpv2-userns-e2e-serial-kubetest2
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:1349]: ^^ Indentation here is wrong

/hold

**Comment by krzyzacy** [comment]
also +1 to `can you combine the presubmits into a single PR?` :-)

**Comment by kannon92** [comment]
@elieser1101 
Thank you for taking this work on. I went ahead and closed the other PRs. Please combine all these presubmits into a single PR.

/hold

**Comment by elieser1101** [comment]
Thanks for the help, sorry for the noise.
Think now everything is in place

@kannon92 @krzyzacy

**Comment by kannon92** [comment]
At a glance this looks good to me.

So we want to add the kubetest2 jobs and leave the nonkubetest ones around,

When will we remove the legacy jobs?

**Comment by kannon92** [comment]
Have you confirmed that the tests that have merged already are working without issue?

**Comment by elieser1101** [comment]
The idea would be to merge these test, then create a PR on which validate all the kubetest2 presubmits works as expected and refactor anything broken.

After we validate these new test consistently work for some period of time(not sure how much) then we can remove the legacy ones?

**Comment by kannon92** [comment]
You merged a canary one a few weeks ago. Have you tested that one to see if it worked?

**Comment by elieser1101** [comment]
3 out of 5 of the merged PR were tested and worked here:
https://github.com/kubernetes/kubernetes/pull/120459

**Comment by krzyzacy** [comment]
I'm fine with adding these jobs since they are optional and non-blocking, we need to be a bit careful when we removing the older jobs and make sure the new ones become blocking and don't accidentally lose coverage

**Comment by krzyzacy** [comment]
(see if @kannon92 has any further concerns, otherwise I'll be happy to approve this bulk and track triaging in https://github.com/kubernetes/kubernetes/issues/127831)

**Comment by elieser1101** [comment]
sure, my plan is keep working on this till test are fully working

**Review by krzyzacy — APPROVED** [review]
/lgtm

**Comment by pohly** [comment]
pull-kubernetes-node-e2e-crio-cgrpv1-dra-kubetest2 and pull-kubernetes-node-e2e-crio-cgrpv2-dra-kubetest2 are not working. Apparently the `--label-filter` argument is passed to `kubetest2`, which fails (`Error: unknown flag: --label-filter`).

cc @klueska @bart0sh

**Pull request merged**